src/appengine/handlers/cron/cleanup.py

```diff
@@ -433,10 +435,8 @@ def mark_issue_as_closed_if_testcase_is_fixed(testcase, issue):
     comment += INTERNAL_INCORRECT_COMMENT
     comment += ' and re-open the issue.'
 
-  issue.comment = comment
   issue.status = 'Verified'
-  issue.open = False
```

Comment:
this is not actually needed, because it's not sent in the actual API request to monorail. the openness of an issue is based on its status value.